modules/geolocation/geolocation.py

```diff
@@ -118,10 +119,18 @@ def lat_lon_from_local(self, x, y):
         return latitude, longitude
 
 
-    def gather_point_pairs(self):
+    def gather_point_pairs(self, cameraResolution, camera3u, camera3v, camera3c, cameraOrigin3o, referencePixels):
```

Comment:
Camera resolution is a constant, so it should be `self.` instead of a parameter.